### PR TITLE
Move version_output & configure_args to metadata

### PR DESCRIPTION
**Pull request by M-Davies**
* Moves the output of version.txt and configure.txt to the main metadata over VersionInfo
* Adds Prefixs to console out's for readability

Closes: #11 
Signed-off-by: Morgan Davies <morgan.davies@ibm.com>

### CODE REVIEW

**Comment by M-Davies** [comment]
[PR is ready for review](https://ci.adoptopenjdk.net/view/work%20in%20progress/job/jdk15-mac-x64-openj9/10/). Adding in documentation updates to account for the changes...

File: pipelines/build/common/openjdk_build_pipeline.groovy [hunk 1]
```diff
@@ -394,38 +396,57 @@ class Build {
...
                 }
 
             }
 
+            // Get full version output
+            String versionPath = "workspace/target/version.txt"
+            context.println "INFO: Attempting to read ${versionPath}..."
+
+            // No try catch on this as we should always have a version.txt file
+            fullVersionOutput = context.readFile(versionPath)
```
Comment (karianna) [inline pipelines/build/common/openjdk_build_pipeline.groovy:429]: I think you'll want some defensive programming here, you never know......

File: pipelines/build/common/openjdk_build_pipeline.groovy [hunk 2]
```diff
@@ -394,38 +396,57 @@ class Build {
...
+            context.println "SUCCESS: version.txt found"
+
+            // Get configure args
+            String configurePath = "workspace/target/configure.txt"
+            context.println "INFO: Attempting to read ${configurePath}..."
+
+            // No try catch on this as we should always have a configure.txt file
+            configureArguments = context.readFile(configurePath)
```
Comment (karianna) [inline pipelines/build/common/openjdk_build_pipeline.groovy:437]: See above comment

File: pipelines/build/common/openjdk_build_pipeline.groovy (hunk 1 above)
Comment (M-Davies) [reply pipelines/build/common/openjdk_build_pipeline.groovy:429]: Done

File: pipelines/build/common/openjdk_build_pipeline.groovy (hunk 2 above)
Comment (M-Davies) [reply pipelines/build/common/openjdk_build_pipeline.groovy]: Done

**Pull request merged**